requirements2.txt

```diff
@@ -1,4 +0,0 @@
-# Python2 needs the concurrent.futures backport.
-# We handle this logic in setup.py but we need
-# a separate file to track this in our requirements file.
-futures==2.2.0
```

Comment:
Yes, I updated the .travis.yml file as well to handle this (and I have an upcoming PR that refactors this so other CIs don't need to be updated).